### PR TITLE
Print out MY OWN numbers

### PR DESCRIPTION
**Pull request by lancedolan**
The idea of this branch is that I will print out my own numbers before miguel's numbers...

### CODE REVIEW

File: firstProject.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 
 console.log("Miguel's favorite color is orange");
 console.log( 'green' ) ;
-console.log("2 34 56");
\ No newline at end of file
+
+//now I'm going to print out MY OWN numbers...
+console. 1 2 3 4
```
Comment (miguel-tamayo) [inline firstProject.js]: The correct way to write this would be console.log("1 2 3 4");

Comment (lancedolan) [reply firstProject.js]: AhA let me fix that

**Comment by lancedolan** [comment]
I fixed the code based on your comments.... Can we merge it??

**Comment by miguel-tamayo** [comment]
yup! It's good to merge

**Pull request merged**